amlb/results.py

```diff
@@ -308,6 +313,16 @@ def save_predictions(dataset: Dataset, output_file: str,
 
         df = df.assign(predictions=preds)
         df = df.assign(truth=truth)
+        if quantiles is not None:
+            quantiles.reset_index(drop=True, inplace=True)
```

Comment:
It is bold to do an inplace operation that alters outer context, let's be safe and avoid inplace operations. (I know that it is probably ok here, but trust me when I say that the nastiest bugs are those involving outer context manipulation caused by inplace operations)